tests/Unit/Helpers/ParallelAlgorithms/LinearSolver/LinearSolverAlgorithmTestHelpers.hpp

```diff
@@ -318,11 +328,13 @@ Phase determine_next_phase(const Phase& current_phase,
 }
 
 template <typename Metavariables>
-using component_list =
-    tmpl::append<tmpl::list<ElementArray<Metavariables>,
-                            observers::ObserverWriter<Metavariables>,
-                            OutputCleaner<Metavariables>>,
-                 typename Metavariables::linear_solver::component_list>;
+using component_list = tmpl::append<
```

Comment:
use push_back or push_front instead of append to add multiple elements to a list